src/cmake/config/win-vs.txt

```diff
@@ -154,18 +154,6 @@ if (USE_AVX2)
         /arch:AVX2                          # Advanced Vector Extensions 2
     )
 endif ()
-if (USE_FMA)
```

Comment:
Maybe we could hide this options for windows?
Or hide them everywhere and decide that AVX2 = AVX2 + FMA + F16C on unix?